src/LLMProviders/chainRunner/__tests__/CopilotPlusParallelExecution.test.ts

```
@@ -0,0 +1,208 @@
+      updateLoadingMessage,
+      originalUserMessage: "question",
+    });
+
+    expect(mockedExecuteCoordinatorFlow).not.toHaveBeenCalled();
+    expect(mockedToolManager.callTool).toHaveBeenCalledWith(tool, { query: "foo" });
+    expect(updateLoadingMessage).toHaveBeenLastCalledWith(LOADING_MESSAGES.DEFAULT);
+    expect(result.toolOutputs).toEqual([{ tool: "webSearch", output: { status: "ok" } }]);
+  });
+});
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)